pkg/reconciler/integration/sink/controller.go

```diff
@@ -99,14 +119,14 @@ func NewController(
 	})
 
 	integrationSinkGK := v1alpha1.SchemeGroupVersion.WithKind("IntegrationSink").GroupKind()
-
-	// Enqueue the JobSink, if we have an EventPolicy which was referencing
-	// or got updated and now is referencing the JobSink.
 	eventPolicyInformer.Informer().AddEventHandler(auth.EventPolicyEventHandler(
 		integrationSinkInformer.Informer().GetIndexer(),
 		integrationSinkGK,
 		impl.EnqueueKey,
 	))
 
+	factory.Start(ctx.Done())
+	factory.WaitForCacheSync(ctx.Done())
```

Comment:
I think until we call the `Informer` method on the `cmCertificateInformer` variable to setup handlers this could fail